codalab/worker/worker_run_state.py

```diff
@@ -262,16 +273,6 @@ def _transition_from_PREPARING(self, run_state):
         else:
             docker_network = self.docker_network_internal.name
 
-        try:
```

Comment:
The "3) Set up container" comment needs to be updated then, because it's not a good comment for just setting the `docker_network`.  Maybe just remove "3)" as a step.

Reply:
That's true. Removed.